gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -716,3 +716,7 @@ api:
 # Scoring
 scoring:
   enabled: false
+
+# Cloud
+cloud:
+  enabled: false
```

Comment:
As there is a default value in the java code (false), this settings will always be set.
And I'm not sure it's useful to write it in the gravitee.yml file. It could confuse users.

And as it is more a internal configuration, I think it should be better to keep it only in the code and not expose it through this file.